app/components/articleSearch/components/requestPaperDialog/index.tsx

```diff
@@ -42,31 +41,24 @@ function validateForm(values: FormState) {
 }
 
 const RequestPaperDialog: React.FunctionComponent<RequestPaperDialogProps> = props => {
-  const { currentUser, location, isOpen, onClose, dispatch, query } = props;
+  const { currentUser, isOpen, onClose, dispatch, query } = props;
   const [email, setEmail] = React.useState('');
   const [isLoading, setIsLoading] = React.useState(false);
 
   async function handleSubmitForm(values: FormState) {
     setIsLoading(true);
 
     const feedbackManger = new FeedbackManager();
-
-    let gaId = '';
-    if (typeof ga !== 'undefined') {
-      ga((tracker: any) => {
-        gaId = tracker.get('clientId');
-      });
-    }
-
-    const href: string = location.pathname;
+    const feedbackDesc = `query : ${query} / comment : ${values.content}`;
 
     try {
-      await feedbackManger.sendFeedback({
-        content: values.content,
+      await feedbackManger.sendTicketToFreshdesk({
         email: values.email,
-        referer: href,
-        userId: currentUser.isLoggedIn ? currentUser.id.toString() : '',
-        gaId,
+        description: feedbackDesc,
+        subject: 'Not include paper : ' + values.email,
+        status: 2,
```

Comment:
![image](https://user-images.githubusercontent.com/9796103/64577791-672daa80-d3b8-11e9-918d-9a3d4dd971d1.png)
I think we should map these values as `enum` or `type` to avoid mistake and easy to understand.